archive/gzip/gzip.go

```diff
@@ -0,0 +1,61 @@
+// Package gzip implements reading and writing of gzip format compressed files.
+// See the compress/gzip package for more details.
+package gzip
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+)
+
+// A Reader is an io.Reader that can be read to retrieve uncompressed data from
```

Comment:
Inconsistent comment style -- other interfaces and structs start with their name without any modifier, so more consistent to remove `A`